src/content/docs/tutorial-kubernetes-learn/tutorial-k8s-intro.mdx

```diff
@@ -0,0 +1,148 @@
+---
+title: Understand your kubernetes system with New Relic
+---
+
+import tutorialNavigator from 'images/kubernetes_screenshot-full_navigator-main.webp'
+
+import tutorialInstall from 'images/kubernetes_screenshot-full_guidedInstall.webp'
+
+import k8sNavigatorAnimation from 'images/kubernetes_screenshot-full_navigator-animated.gif'
+
+import apmApmCompleteSummaryPage from 'images/apm_screenshot-full_apm-complete-summary-page-.webp'
+
+import infraComplete from 'images/infrastructure_screenshot-full_infra-main.webp'
+
+
+A Kubernetes system is inherently complex and Kubernetes as a whole has a reputation as intricate and challenging to grasp. Understanding Kubernetes as a concept is a complex task by itself and wanting to understand an indivdual Kubernetes system only adds more complextiy. How do you troubleshoot ephemeral containers that spin up and down before you can even access them? How do you understand the health of your system as a whole if you have hundreds of containers orchestrated at a time?
+
+In this tutorial series, we will explore how you can monitor your Kubernetes system with New Relic and how it can streamline your understanding of the technology.
```

Comment:
I think this needs to be stronger and more compelling. If I'm gonna spend my time with your words, what do I get out of it?

Also, drop the "will" and the "we".

We're not doing the tutorial. The reader (you) is.

If you're super attached to the first person plural, you might go with something like: "Our goal with this tutorial is to help you...etc etc"